lib/realtime_web/channels/user_socket.ex

```diff
@@ -4,9 +4,25 @@ defmodule RealtimeWeb.UserSocket do
   @impl true
   def init(state) when is_tuple(state) do
     Process.flag(:max_heap_size, max_heap_size())
+    Process.send_after(self(), {:measure_traffic, 0, 0}, measure_traffic_interval_in_ms())
     Phoenix.Socket.__init__(state)
   end
 
+  @impl true
+  def handle_info(
+        {:measure_traffic, previous_recv, previous_send},
+        {_, %{assigns: assigns, transport_pid: transport_pid}} = state
+      ) do
+    tenant_external_id = Map.get(assigns, :tenant)
+
+    %{send_delta: send_delta, recv_delta: recv_delta} =
+      collect_traffic_telemetry(transport_pid, tenant_external_id, previous_recv, previous_send)
+
+    Process.send_after(self(), {:measure_traffic, recv_delta, send_delta}, measure_traffic_interval_in_ms())
```

Comment:
Should we be sending delta 🤔 ? We should be sending the actual value I think? 

Example:

* Websocket starts with {0, 0}.
* First stats are {10, 5}. We emit the delta which is 10 - 0, 5 - 0. Schedule message with {10, 5}
* Second stats are {25, 7}. We emit the delta which is 25 - 10, 7 - 5. Schedule message with {25, 7}